src/Controls/src/Core/NavigationPage/NavigationPageToolbar.cs

```diff
@@ -265,7 +265,7 @@ void ApplyChanges(NavigationPage navigationPage)
 		}
 
 		Color GetBarTextColor() => _currentNavigationPage?.BarTextColor;
-		Color GetIconColor() => (_currentPage != null) ? NavigationPage.GetIconColor(_currentPage) : null;
+		Color GetIconColor() => NavigationPage.GetIconColor(_currentPage) ?? NavigationPage.GetIconColor(_currentNavigationPage);
```

Comment:
"The icon color was respected when setting this value directly for the content page NavigationPage.IconColor="Red", but if it was not set there then it has not been picked from its parent navigation page"

Only happens on Android?

Reply:
Yes, but iOS doesn't have this three-dots overflow icon